sonic_data_client/non_db_client.go

```diff
@@ -63,6 +63,10 @@ var (
 			path:    []string{"OTHERS", "proc", "stat"},
 			getFunc: dataGetFunc(getProcStat),
 		},
+		{ // Get host uptime
+			path:    []string{"OTHERS", "platform", "sysuptime"},
```

Comment:
Shouldn't the path be `proc/uptime`?

The other `proc` based paths have `proc` prefix. This is in the same category with them.

Reply:
I think the dataset `system uptime` is depended on platform, just like the dataset `CPU utilization` at line 43.

Reply:
@yozhao101 "/proc/uptime" should not be dependent on platform. Do you have any reference here?

Reply:
I referred the dataset of `CPU utilization`. Did you mean we should put system uptime under the query of `proc`, right?

Reply:
yes let's put it as `/proc/uptime ` prefix

Reply:
updated! Please have a review!